lib/ruby/requirement.ts

```diff
@@ -16,30 +16,37 @@
 
 import { Version } from './version';
```

Comment:
Maybe a type import only ?
```suggestion
import type { Version } from './version';
```

Reply:
Won't work as it's a constructor